### PR TITLE
Fix interface of `TxDelegate` and `TxUndelegate`

### PR DESCRIPTION
**Pull request by TomTaehoonKim**

### CODE REVIEW

File: common/src/types.rs [hunk 1]
```diff
@@ -115,19 +115,14 @@ pub enum ExtraAgendaTransaction {
 
 #[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Clone)]
 pub struct TxDelegate {
-    pub delegator: PublicKey,
-    pub delegatee: PublicKey,
-    /// Whether to delegate the governance voting power too.
-    pub governance: bool,
+    pub transaction_data: DelegationTransactionData,
```
Comment (junha1) [inline common/src/types.rs:118]: I think just `data` is better.

File: common/src/types.rs [hunk 2]
```diff
@@ -115,19 +115,14 @@ pub enum ExtraAgendaTransaction {
...
     pub proof: TypedSignature<DelegationTransactionData>,
-    pub timestamp: Timestamp,
 }
 
 #[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Clone)]
 pub struct TxUndelegate {
-    pub delegator: PublicKey,
+    pub transaction_data: UndelegationTransactionData,
```
Comment (junha1) [inline common/src/types.rs:124]: ditto

File: common/src/types.rs (hunk 1 above)
Comment (TomTaehoonKim) [reply common/src/types.rs]: Fixed.

File: common/src/types.rs (hunk 2 above)
Comment (TomTaehoonKim) [reply common/src/types.rs]: Fixed.

**Review by junha1 — APPROVED** [review]

**Pull request merged**